doc/guides/collaborator-guide.md

```diff
@@ -469,6 +469,11 @@ code. If you wish to create the token yourself in advance, see
 
 ### Technical HOWTO
 
+Infrequently, it is necessary to manually perform the steps required to land a
+pull request rather than rely on `git-node`.
+
+<details>
+
```

Comment:
nit(feel free to ignore!): we can make this a little prettier by making this:

```
<details>
<Summary>Manual Landing Steps</Summary>

</details>
```

or similar

Reply:
Added. Thanks!